libs/designsystem/src/lib/scss/base/_typography.scss

```diff
@@ -41,56 +41,78 @@ kirby-app .page-title {
   line-height: normal;
 }
 
+$apple-headline-size: 17px;
+$apple-body-size: 17px;
+
 h1,
 .h1,
 .kirby-text-xlarge {
   font-size: font-size('xl');
+  @supports (font: -apple-system-headline) {
```

Comment:
To avoid setting `font` & `font-family` for every font style; consider setting `font` & `font-family` once by creating a selector targeting the appropriate elements if it makes sense. Alternatively it could perhaps be packaged in a mixin somehow together with font-size?